operations/phrases/dogs.js

```diff
@@ -0,0 +1,22 @@
+const axios = require('axios');
+
+module.exports = dogPhrases = async (args, message) => {
+  const supportedDogs = [
+    'dachsund', 'pug', 'schipperke', 'shiba', 'whippet'
+  ];
+
+  let dogFound = null;
+  supportedDogs.forEach((dog) => {
+    if (args.includes(dog)) {
+      dogFound = dog;
+    }
+  });
+
+  if (dogFound) {
+    const dog = await axios.get(`https://dog.ceo/api/breed/${dogFound}/images/random`);
+    message.channel.send(`${dogFound} acquired`, { files: [ dog.data.message ] });
```

Comment:
would you be willing to write some error handling for this? like check if `dog.data` axios response is 200 and exists and if not send some kind of error to chat instead.

Reply:
Yes! Error handling is important